share/availability/window.go

```diff
@@ -0,0 +1,24 @@
+package availability
+
+import "time"
+
+const (
+	RequestWindow = 30 * 24 * time.Hour
+	StorageWindow = RequestWindow + time.Hour
+)
+
+type Window time.Duration
```

Comment:
TBH I don't get it why do we need this type. Looks like `time.Duration` is more than enough in all cases. What I'm missing?

Reply:
nodebuilder :( I can move the definition to nodebuilder in this PR I guess which is what @walldiss wanted

Reply:
moved it in https://github.com/celestiaorg/celestia-node/pull/3906/commits/1e260c3179c88f0491885fe4dcee08034ff15047